src/library/basesqltablemodel.h

```diff
@@ -139,6 +139,23 @@ class BaseSqlTableModel : public QAbstractTableModel, public TrackModel {
         }
         int m_column;
         Qt::SortOrder m_order;
+        
+        friend QTextStream& operator >> (QTextStream &in, SortColumn &sc) {
```

Comment:
But if we save the column name as a string this should solve the problem. It is true that there are no sanity checks but this can only be a problem if we change the name or remove a column that the user has previously sorted but once she tries to sort again and restarts mixxx the problem solves (when I changed from integers to strings I had this problem and I forced sorting and restarted and then it worked fine again)